2nd-gen/packages/swc/.storybook/DocumentTemplate.mdx

```diff
@@ -28,26 +29,18 @@ This template will likely be going away in the future, but for now it's here to
         <Description />
         <Primary />
         <Controls />
-
-        ### third level
-
-        hello
-
-        #### fourth level
-
-        bananas my friends
     </sp-tab-panel>
     <sp-tab-panel value="usage" style={{ flexDirection: 'column' }}>
-            variations
-
+        <SpectrumDocs tag="usage" />
     </sp-tab-panel>
     <sp-tab-panel value="accessibility" style={{ flexDirection: 'column' }}>
-            Heres some shit about accessibility
+        <SpectrumDocs tag="a11y" />
     </sp-tab-panel>
     <sp-tab-panel value="api" style={{ flexDirection: 'column' }}>
-            <ArgTypes />
+        <ArgTypes />
```

Comment:
I think this is where we have an example with controls and code, so the API is interactive.